tests/e2e_test.go

```diff
@@ -283,3 +286,28 @@ func TestUploadNoAPIKeyInOutput(t *testing.T) {
 		t.Errorf("CLI output contained API key when it should have squelched it")
 	}
 }
+
+func TestWatch(t *testing.T) {
+
+	td := t.TempDir()
+	cmd := exec.Command("../rekor-server", "watch", "--interval=1s")
```

Comment:
`s|"../rekor-server"|server|`

Reply:
I don't think so... We compile it as "rekor-server" above, and the test directory here is set to test/, so we need to go back up one.

Reply:
Oh sorry I should have been more specific! The string replacement was to use the `server` constant var defined below as I assumed that's what you intended and I didn't see the constant used anywhere else.